### ISSUE TITLE
FAQ - how does flatpak compare to other app frameworks?

### ISSUE THREAD

**Issue by allanday** [flatpak/flatpak.github.io#228]
From issue #58, we probably ought to add the following to the FAQ:

 * How does flatpak compare to other application sandboxing technologies: zeroinstall, glick, snap, ... ?
 * How does flatpak compare to other packaging technologies: rpm, dep,... ?

**Comment by allanday** [comment]
We should consider whether this should go in the docs instead.

**Comment by TingPing** [comment]
I think it is very hard to get comparisons to alternatives right. It would be easy to compare to traditional formats though.

**Comment by marceloslacerda** [comment]
Yeah specially since both flatpak and the alternatives are undergoing heavy development, any comparison would quickly drift apart from what's actually implemented on each tool.
